src/components/Previews.tsx

```diff
@@ -158,14 +176,26 @@ export const ItemLabelPreview: React.FC<{
             break
     }
 
+    const [randomBackgroundImageIndex, setBackgroundImageIndex] = useState(
+        Math.floor(Math.random() * backgroundImages.length)
+    )
+
+    window.setTimeout(
+        () => {
+            setBackgroundImageIndex(
+                Math.floor(Math.random() * backgroundImages.length)
+            )
+        },
+        Math.floor(Math.random() * (60000 - 30000 + 1)) + 30000
+    )
```

Comment:
This is going to create a new `setTimeout` every time this component is rendered and I think if you leave the page open long enough you'll get a Christmas tree 🎄 

https://github.com/user-attachments/assets/4982cb8b-9693-495b-98fc-27ace5bd7ff0

Reply:
I think you'd want to tell React to only run this once when the component is first rendered and then use [setInterval](https://developer.mozilla.org/en-US/docs/Web/API/Window/setInterval) to change the background every so often (if that's what you wanted to do)

```suggestion
    useEffect(() => {
        const interval = window.setInterval(
            () => {
                setBackgroundImageIndex(
                    Math.floor(Math.random() * backgroundImages.length)
                )
            },
            Math.floor(Math.random() * (60000 - 30000 + 1)) + 30000
        )

        return () => {
            clearInterval(interval)
        }
    }, [])
```

Reply:
Oh hmm i thought that the re-render would get scoped to the preview and the set-timeout would have the expected effect even if we did hi-jack the re-render to trigger the next one. 
:( guess i still don't understand enough reactjs